src/host.js

```diff
@@ -94,7 +94,7 @@ Host._createIFrame = function createIFrame(src, frameId, height, allowFullScreen
 };
 
 Host._tryGetOrigin = function tryGetOrigin(url) {
-	if (url && url.indexOf('//') === 0) {
+	if (url && url.indexOf('//') === -1) {
```

Comment:
Should we add another if statement for `url && url.indexOf('//') === -1` case?

Reply:
@omsmith We're using ifrau in this context:
https://github.com/Brightspace/what-the-faq/blob/master/Projects/ManagerRole/Local-dev-setup.md#setup-local-fra
and if we omit the protocol altogether (also no //) things just don't work, so it seems logical to add an extra case were when the user does not add a protocol altogether, agree?

Reply:
Sorry, don't understand why that should change anything.

What are you trying to pass in as a `src` to host?